### ISSUE TITLE
Gandi v5 DNS management

### ISSUE THREAD

**Issue by jean-gui** [brianpcurran/gandi-automatic-dns#20]
Hi,

I'm using the beta version of Gandi v5, its new nameservers and its new way of managing zones. It seems that gandi-automatic-dns isn't compatible with it as I get the following when running the command:
```
$ ./gad -a ***** -d ***** -r ****** -e
Initial flags:

apikey = *****
domain = *****
records = *****
record_type = A
ip_regex = \([0-9]*\.[0-9]*\.[0-9]*\.[0-9]*\)

IP information:

ext_ip_method = OpenDNS
ext_ip = *****

RPC call to methodName = domain.info

Sending XML-RPC message tmp_message =

POST /xmlrpc/ HTTP/1.0
User-Agent: Gandi Automatic DNS shell script/0.4
Host: rpc.gandi.net:443
Content-Type: text/xml
Content-Length: 270

<?xml version="1.0"?>
<methodCall>
  <methodName>domain.info</methodName>
  <params>
    <param>
      <value><string>*****</string></value>
    </param>
    <param>
      <value><string>*****</string></value>
    </param>
  </params>
</methodCall>

zone_id = 

No zone_id returned. This is expected with Gandi's test API. Use gad's -t flag for testing.
```

**Comment by brianpcurran** [comment]
That's correct, gad doesn't currently support Gandi v5/LiveDNS. There aren't currently any plans to support it, but I may get around to it sometime in the future. In the meantime if you need gad I'd suggest going back to Gandi's classic DNS if it's possible.

**Comment by brianpcurran** [comment]
FWIW I'm now using LiveDNS for my domains, so gad may get an update in the near future to support it...stay tuned.

**Comment by brianpcurran** [comment]
The [1.0.0 branch](https://github.com/brianpcurran/gandi-automatic-dns/tree/1.0.0) appears to be working with Gandi v5/LiveDNS. I'm going to hold off on merging to master until I can do some more tests and clean up the code a little bit. All you should need to do to use it is provide a new LiveDNS API key, and pass the `-5` flag to the script.

**Comment by brianpcurran** [comment]
I just merged the 1.0.0 branch into master and tagged the [1.0.0 release](https://github.com/brianpcurran/gandi-automatic-dns/releases/tag/v1.0.0). I'm going to close this issue out, but let me know if you have any issues with the new v5/LiveDNS support.
